docs/tutorials/vertical-fl.md

```diff
@@ -0,0 +1,113 @@
+
+Afterwards, we can continue with regular training loop:
+- **Forward pass in contributors** - all contributors, and optionally host, performs forward pass on their part of the model with features they own
+- **Intermediate outputs transfer** - all outputs from previous step are sent to the host that performs an aggregation (for simplicity sake we make use of mean operation)
+- **Loss computation** - host performs either forward pass on its part of network or just passes aggregated outputs of previous step through an activation function followed by loss computation
+- **Gradients computation** - if host own some part of the network, it performs backward pass, followed by computing gradients w.r.t inputs in all cases
+- **Gradient transfer** - all contributors, and optionally host, receives gradients w.r.t. their intermediate outputs
+- **Backward pass** - gradients are used to perform backward pass and update the network weights
+
+<br/><br/>
+<div align="center">
+    <img src="../pics/vfltrainingloop.png" alt="Vertical federated learning training loop" width="400">
+</div>
+
+### Steps to launch
+1. If you are not using the quickstart setup, adjust the config file  `config.yaml` in `examples/pipelines/<example-name>/` to match your setup.
+
+2. Submit the experiment by running:
+
+   ```bash
+   python ./examples/pipelines/<example-name>/upload_data/submit.py --config examples/pipelines/<example-name>/config.yaml --workspace_name "<workspace-name>" --resource_group "<resource-group-name>" --subscription_id "<subscription-id>"
+   ```
+
+   > Note: You can use --offline flag when running the job to just build and validate pipeline without submitting it.
+
+    :star: you can simplify this command by entering your workspace details in the file `config.yaml` in this same directory.
+
+
+## Tips and pitfalls
+1. **Vertical Federated Learning comes at a cost**
+    There is significant overhead when launching vertical federated learning due to heavy communication among participants. As we can see in the training loop there are two transfers per each mini-batch. One for forward pass outputs, one for gradients. This means that the training may take longer than expected.
+2. **Intersection and entity alignment**
+   The samples needs to be aligned across participants ahead of the training after we created set intersection of samples that are present on all involved parties. This process can reveal information to other entities that we may want to keep private. Fortunately there are **private set intersection** methods available out there that come to rescue.
+3. **Communication encryption**
+    Even though the intermediate outputs and gradients are not raw data, they still have been inferred using possibly private data. Therefore, it's good to use some sort of security when communicating the data to untrusted parties outside of Azure (e.g. differential privacy).
```

Comment:
Be explicit about "some sort of security when communicating the data". Do you mean "encryption"? If so, then write so. I'm not quite sure what DP is doing here, it doesn't really have to do with communication encryption, it protects from the model to be able to regurgitate data